README.md

```diff
@@ -7,59 +7,411 @@
+    // the settlement transfers we've received.
+    ctx.state.maxUnsecured = opts.maxUnsecured
+  },
+
+  handleIncomingPrepare: async function (ctx, transfer) {
+    const incoming = await ctx.transferLog.getIncomingFulfilledAndPrepared() 
+
+    // Instead of getting the best claim, we're getting the sum of all our
+    // incoming settlement transfers. This tells us how much incoming money has
+    // been secured.
+    const amountReceived = await ctx.state.incomingSettlements.getIncomingFulfilledAndPrepared()
+
+    // The peer can default on 'incoming - amountReceived', so we want to limit
+    // that amount.
+    const exceeds = new BigNumber(incoming)
+      .subtract(amountReceived)
+      .greaterThan(ctx.state.maxUnsecured)
+
+    if (exceeds) {
+      throw new NotAcceptedError(transfer.id + ' exceeds max unsecured balance')
+    }
+  },
+
+  // Even though this function is designed for creating a claim, we can
+  // very easily repurpose it to make a payment for settlement.
+  createOutgoingClaim: async function (ctx, outgoingBalance) {
+    // If a new max value is set, the maxValueTracker returns the previous max
+    // value. We tell the maxValueTracker that we're gonna pay the entire
+    // outgoingBalance we owe, and then look at the difference between the last
+    // balance and the outgoingBalance to determine how much to pay.
+    // If we've already paid out more than outgoingBalance, then it won't be the
+    // max value. The maxValueTracker will return outgoingBalance as the result,
+    // and outgoingBalance - outgoingBalance is 0. Therefore, we send no payment.
+    const lastPaid = ctx.state.amountSettled.setIfMax({ value: outgoingBalance, data: null })
+    const diff = outgoingBalance - lastPaid.value
+
+    if (!diff) {
+      return
+    }
+
+    // We take the transaction ID from the payment we send, and give it as an
+    // identifier so out peer can look it up on the network and verify that we
+    // paid them. Another approach could be to return nothing from this
+    // function, and have the peer automatically track all incoming payments
+    // they're notified of on the settlement ledger.
+    const txid = Network.makePaymentToPeer(diff)
+
+    return { txid }
+  },
+
+  handleIncomingClaim: async function (ctx, claim) {
+    const { txid } = claim
+    const payment = await Network.getPayment(txid)
+
+    if (!payment) {
+      return
+    }
+
+    // It doesn't really matter whether this is fulfilled or not, we just need
+    // it to affect the incoming balance so we know how much has been received.
+    // We use the txid as the ID of the incoming payment, so it's impossible to
+    // apply the same incoming settlement transfer twice.
+    await ctx.state.incomingSettlements.prepare({
+      id: txid,
+      amount: payment.amount
+    }, true) // isIncoming: true
+  }
+
+}
 ```
+
+## Backend API, with Extensions for Payment Channels
```

Comment:
It's passed in as the `_backend` parameter, or if the `_store` parameter is specified instead, a backend will be constructed on top of the store. I'm still considering some changes on how the payment channel backend interacts with the constructor and the prefix and such, but I'll write a section on the plugin constructor once I've decided.